subsys/mgmt/fmfu/src/fmfu_mgmt.c

```diff
@@ -108,21 +108,6 @@ static int unpack(struct mgmt_ctxt *ctxt, struct firmware_packet *packet,
 	return offset + packet->data_len;
 }
 
-static int get_mgmt_err_from_modem_ret_err(int err)
```

Comment:
Does `nrf_modem_full_dfu_bl_write` no longer return an error message of any use? With this proposal what ever is returned from this function is "overwritten" with `MGMT_ERR_EBADSTATE`